waspc/src/Generator/ServerGenerator.hs

```diff
@@ -120,14 +123,20 @@ waspNpmDeps =
       ("debug", "~2.6.9"),
       ("express", "~4.16.1"),
       ("morgan", "~1.9.1"),
-      ("@prisma/client", "2.22.1"),
+      ("@prisma/client", "2.21.0"),
```

Comment:
Watch out, this change shouldn't have happened!

Reply:
My fault, fixed!